libs/application/templates/parental-leave/src/forms/Prerequisites.ts

```diff
@@ -0,0 +1,165 @@
+import get from 'lodash/get'
+import {
+  buildCustomField,
+  buildDataProviderItem,
+  buildExternalDataProvider,
+  buildForm,
+  buildMultiField,
+  buildRadioField,
+  buildSection,
+  buildSubmitField,
+  buildSubSection,
+  Form,
+  FormModes,
+} from '@island.is/application/core'
+
+import { parentalLeaveFormMessages } from '../lib/messages'
+import Logo from '../assets/Logo'
+
+export const PrerequisitesForm: Form = buildForm({
+  id: 'ParentalLeaveDraft',
```

Comment:
Should probably have a different id than the main template

Reply:
Copy paste, my bad 💩  thanks!